src/test/kotlin/fi/metatavu/muisti/api/test/functional/builder/TestBuilder.kt

```diff
@@ -0,0 +1,78 @@
+package fi.metatavu.muisti.api.test.functional.builder
+
+import fi.metatavu.jaxrs.test.functional.builder.AbstractAccessTokenTestBuilder
+import fi.metatavu.jaxrs.test.functional.builder.AbstractTestBuilder
+import fi.metatavu.jaxrs.test.functional.builder.auth.AccessTokenProvider
+import fi.metatavu.jaxrs.test.functional.builder.auth.AuthorizedTestBuilderAuthentication
+import fi.metatavu.jaxrs.test.functional.builder.auth.KeycloakAccessTokenProvider
+import fi.metatavu.muisti.api.client.infrastructure.ApiClient
+import fi.metatavu.muisti.api.test.functional.builder.auth.TestBuilderAuthentication
+import fi.metatavu.muisti.api.test.functional.mqtt.TestMqttClient
+import fi.metatavu.muisti.api.test.functional.settings.MqttTestSettings
+import org.eclipse.microprofile.config.ConfigProvider
+import org.slf4j.LoggerFactory
+import java.io.IOException
+
+/**
+ * TestBuilder implementation
+ *
+ * @author Antti Leppä
+ */
+class TestBuilder(private val config: Map<String, String>) : AbstractAccessTokenTestBuilder<ApiClient>() {
+
+    private var admin: TestBuilderAuthentication? = null
```

Comment:
this could be initialized directly here, instead of using separate function

Reply:
done